scripts/variantstore/wdl/extract/Dockerfile

```diff
@@ -1,14 +1,17 @@
 FROM gcr.io/google.com/cloudsdktool/cloud-sdk:349.0.0
 
 # Copy the application's requirements.txt and run pip to install
-ADD requirements.txt /app/requirements.txt
+COPY requirements.txt /app/requirements.txt
 RUN pip install -r /app/requirements.txt
 RUN apt-get update && apt-get -y upgrade && apt-get -y install bcftools
 
-# Add the application source code.
-ADD create_cohort_extract_data_table.py /app
-ADD create_variant_annotation_table.py /app
-ADD extract_subpop.py /app
+# Copy the application source code.
+COPY create_cohort_extract_data_table.py /app
+COPY create_variant_annotation_table.py /app
+COPY extract_subpop.py /app
+COPY populate_alt_allele_table.py /app
+COPY alt_allele_positions.sql /app
+COPY alt_allele_temp_function.sql /app
```

Comment:
why the switch?

Reply:
[docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) said it was the preferred method